### PR TITLE
feat: add scid to summary output

### PR DESCRIPTION
**Pull request by m-schmoock**
Improves some minor output issues:

 - Adds short channel id, so it can be copy and pasted for rebalancing
 - renames `channels_key` to `channels_flags`
 - replace `=` with `:` in `channels_flags`, looks better in -H or JSON output.
 - changes flags round rackets (PO) to squace brackets [PO]
 - Always outputs flags even if not set `[__]` so things are always aligned
 - Removes the colon ':' and always prints a space
   this also helps when copy and pasting in a terminal

TLDR: Output will now look like this (with correct special char alignment which GitHub mess up):
```
channels_flags=P:private O:offline
channels=

    ├──────────────────┼───┤                    1511185x45x0   [__] testnet1-lnd.zaphq.io
├──────────────────────╢                        1511189x47x0   [P_] CoinGate
 ├─────────────────────┼┤                       1511205x23x0   [_O] lnd.fun
```

### CODE REVIEW

**Comment by m-schmoock** [comment]
Since its only output formatting I did it all in one commit.
If you want it separately, I can forcepush update the PR.

regargs,
Michael

**Comment by cdecker** [comment]
ACK c58cbd6e68309d5c08503fc2cd441b564c11d3f2
